### PR TITLE
Update CLI help for `provider` command

### PR DESCRIPTION
**Pull request by irmatov**

### CODE REVIEW

File: cmd/provider.go [hunk 1]
```diff
@@ -4,12 +4,15 @@ import (
 	"github.com/spf13/cobra"
 )
 
+var providerHelpMsg = "Provider command that unifies provider subcommands."
```
Comment (yevgenypats) [inline cmd/provider.go:7]: "top-level command to interact with providers"

File: cmd/provider.go [hunk 2]
```diff
@@ -4,12 +4,15 @@ import (
...
 var providerCmd = &cobra.Command{
 	Use:   "provider [subcommand]",
-	Short: "Provider command that unifies provider subcommands.",
-	Long: `Examples:
-# Download all providers
-./cloudquery provider download
+	Short: providerHelpMsg,
+	Long:  providerHelpMsg,
+	Example: `
+  # Downloads all providers mentioned in the configuration file:
```
Comment (yevgenypats) [inline cmd/provider.go:14]: `Downloads all providers specified in config.hcl:`

File: cmd/provider_download.go [hunk 3]
```diff
@@ -11,13 +11,15 @@ import (
 	"github.com/spf13/viper"
 )
 
+var providerDownloadHelpMsg = "Downloads all providers mentioned in the configuration file."
```
Comment (yevgenypats) [inline cmd/provider_download.go:14]: `Downloads all providers specified in config.hcl`

File: cmd/provider_download.go [hunk 4]
```diff
@@ -11,13 +11,15 @@ import (
...
 	Use:   "download",
-	Short: "Downloads all providers",
-	Long: `Downloads all required providers.
-	
-Examples:
-./cloudquery provider download
+	Short: providerDownloadHelpMsg,
+	Long:  providerDownloadHelpMsg,
+	Example: `
+  # Downloads all providers mentioned in the configuration file:
```
Comment (yevgenypats) [inline cmd/provider_download.go:21]: `Downloads all providers specified in config.hcl`

**Review by yevgenypats — CHANGES_REQUESTED** [review]
Looks good. A few wording changes suggestions.

File: cmd/provider.go (hunk 1 above)
Comment (irmatov) [reply cmd/provider.go:7]: Updated.

File: cmd/provider_download.go (hunk 3 above)
Comment (irmatov) [reply cmd/provider_download.go:14]: Updated.

File: cmd/provider_download.go (hunk 4 above)
Comment (irmatov) [reply cmd/provider_download.go:21]: Updated.

**Pull request merged**